### PR TITLE
Rework test function, and disable expensive tests on CI

### PR DESCRIPTION
**Pull request by Meakk**

### CODE REVIEW

**Comment by Meakk** [comment]
@mwestphal Randomly failing tests because of HDRI computation is annoying.  
I plan to do something about it for 1.3 but for now I suggest to disable them completely on the CI.  
Let me know what you think.

**Comment by mwestphal** [comment]
not random fail, random timeouts.

**Comment by mwestphal** [comment]
not ok imo, just disable on windows, on linux it is fine.

or maybe disable for vtk 9.0

**Comment by Meakk** [comment]
what about this @mwestphal ?

File: cmake/testing.cmake [hunk 1]
```diff
@@ -196,20 +196,16 @@ if(VTK_VERSION VERSION_GREATER 9.0.1)
...
 
   f3d_test_interaction(TestInteractionHDRIMove suzanne.ply "300,300" "--hdri=${CMAKE_SOURCE_DIR}/data/testing/palermo_park_1k.hdr") #Shift+MouseRight;
-  set_tests_properties(TestInteractionHDRIMove PROPERTIES LABELS "HDRI")
 
   # Test exit hotkey
   f3d_test_interaction_no_baseline(TestInteractionSimpleExit cow.vtp "300,300") #Escape;
-  set_tests_properties(TestInteractionSimpleExit PROPERTIES PASS_REGULAR_EXPRESSION "Interactor has been stopped, no rendering performed")
+  if(F3D_ENABLE_LONG_TIMEOUT_TESTS)
+    set_tests_properties(TestInteractionSimpleExit PROPERTIES PASS_REGULAR_EXPRESSION "Interactor has been stopped, no rendering performed")
+  endif()
```
Comment (mwestphal) [inline cmake/testing.cmake:208]: not sure to follow this

Comment (Meakk) [reply cmake/testing.cmake:208]: if `F3D_ENABLE_LONG_TIMEOUT_TESTS` is disabled, `TestInteractionSimpleExit` test target does not exist and produce a CMake error

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -293,6 +292,7 @@ jobs:
       run: >
         cmake ./
         -DBUILD_TESTING=ON
+        -DF3D_ENABLE_LONG_TIMEOUT_TESTS=ON
```
Comment (mwestphal) [inline .github/workflows/ci.yml:295]: Also enable long timeout for standard non-macos, non vtk9.0 build

**Comment by mwestphal** [comment]
you have a failing test

File: CMakeLists.txt [hunk 3]
```diff
@@ -10,6 +10,7 @@ string(TIMESTAMP F3D_BUILD_DATE "%Y-%m-%d %H:%M:%S")
 
 # CMake variables
 option(BUILD_TESTING "Build the tests" OFF)
+cmake_dependent_option(F3D_ENABLE_LONG_TIMEOUT_TESTS "Run long timeout tests" OFF "BUILD_TESTING" ON)
```
Comment (mwestphal) [inline CMakeLists.txt:13]: - Run
+ Enable 

?

**Review by mwestphal — APPROVED** [review]
+1 pending small comment

**Pull request merged**